src/libs/Console/index.ts

```diff
@@ -44,11 +58,15 @@ function logMessage(args: unknown[]) {
 
 /**
  * Override the console.log function to add logs to the store
+ * Log only in production environment to avoid storing large logs in development
  * @param args arguments passed to the console.log function
  */
 // eslint-disable-next-line no-console
 console.log = (...args) => {
-    logMessage(args);
+    if (shouldStoreLogs && CONFIG.ENVIRONMENT !== CONST.ENVIRONMENT.DEV) {
```

Comment:
Coming from the slack thread https://expensify.slack.com/archives/C01GTK53T8Q/p1708081810577189?thread_ts=1703982689.203629&cid=C01GTK53T8Q

> ah, we can’t entirely skip adding a log to onyx here, because this override is needed when e.g. console.log is used in the callback function of the executed arbitrary code. I opened this PR and added an environment check, tested it and works fine.


That means this piece of code won't work?

```js
console.log = (...args) => {
    originalConsoleLog.apply(console, args);
};
```